src/elements/content-sidebar/activity-feed/task-new/AvatarGroupAvatar.scss

```diff
@@ -0,0 +1,20 @@
+@import '../../../common/variables';
+@import './mixins';
+
+$bcs-AvatarGroupAvatar-avatar-size: 32px;
```

Comment:
Curious about component-level mixins for future use. One practice we could consider is adding variables in the components for `import`ing and re-use in features elsewhere. 

`$bcs-AvatarGroupAvatar-avatar-size: $bdl-Avatar-size;`

Reply:
We should move the support for badging into the avatar component itself, IMO. Also refactor it to support multiple sizes instead of just 32px and 44px.